setup_cloudant.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 function getCredentialsPath {
-    for p in InfiniteScrollingwithCloudantNoSQLforiOS*/ ; do
+    for p in iosinfinitescrollingcloudant*/ ; do
```

Comment:
This change might need to be upstreamed to the responsible code generator?

@joyychang

Reply:
@christiancompton I'm not sure exactly what this typo is, but it looks important. Will you apply it upstream?